Neos.Neos/Resources/Private/Templates/Module/Administration/Users/Index.html

```diff
@@ -26,7 +26,31 @@
 						<tr>
 							<th><neos:backend.translate id="users.name" source="Modules" package="Neos.Neos"/></th>
 							<th><neos:backend.translate id="users.accountsAndRoles" source="Modules" package="Neos.Neos"/></th>
-							<th><neos:backend.translate id="users.lastLogin" source="Modules" package="Neos.Neos"/></th>
+							<th>
+								<f:if condition="{sortBy} === 'LastLoggedIn'">
```

Comment:
With this it is not possible anymore to sort the users by account after sorting it by date. We should add the same options for account as well.